### DISCUSSION TITLE
Can get hit but cannot hit back in claims

### DISCUSSION THREAD

**Discussion by NivagonWaffen** [GriefPrevention/GriefPrevention#1374] [Q&A] [answered]
Hello there. I have been having problems on a server that I play on that uses GriefPrevention. There has been a player going around that can damage and kill people in claims but the claim owners cannot fight back nor deal damage. I guess I am asking if there is a way to fix this or a reason for it?

Possible Relevant Info : 

The player is OPed
Sieges are disabled
You cannot deal any damage but they can hit you

**Comment by bigpresh** [comment, marked as answer]
If they're opped, they will bypass GP's checks.

I would suggest that, if the staff on the server are abusing their privileges to mess with players in that way, it's probably not a server you'd want to play on, and you should play somewhere with more responsible staff.
